packages/lib/.storybook/theme.js

```diff
@@ -1,21 +1,21 @@
 import { create } from '@storybook/theming';
 
 export default create({
-  base: 'dark',
-  brandTitle: 'Adyen Web Components',
-  brandTarget: '_self',
+    base: 'dark',
```

Comment:
Same here, styling changes just make it harder to review the PR and focus on the changes.

Reply:
Is it a 2 space vs 4 space misalignment in the linter?